tests/test_reader.py

```diff
@@ -1,9 +1,18 @@
 import libvalkey
 import pytest
 
-@pytest.fixture()
-def reader():
-  return libvalkey.Reader()
+@pytest.fixture(params=[True, False])
+def reader(request):
+  class TestReader(libvalkey.Reader):
```

Comment:
Why is this wrapper needed?

Reply:
Having `listOnly` variable in a class that comes from C extension doesn't expose `listOnly` as a class property. And we don't necessarily want it to be exposed to the public.

However, in case of these tests we do want to keep track whether `Reader` was built with `listOnly=True` or not, because `reader` now became a parametrized fixture. I  checked that simple `request.param` will not work in this case and decided to go with wrapper instead.

Reply:
I think that it might be a good idea to make it part of the Reader public API. listOnly is already a public parameter set by the user and it might be useful to be able to check it later, just like you did in the tests.

Reply:
This one is fixed